exercises/supervised-regression/ex_rnw/ex_regr_losses.Rnw

```diff
@@ -0,0 +1,75 @@
+In this exercise, we will examine loss functions for regression problems 
+somewhat more in-depth.
+
+<<echo=FALSE, message=FALSE, fig.align="left", fig.height=2.5, fig.width=4>>=
+
+set.seed(123L)
+x <- runif(100L)
+y <- 0.2 + 0.8 * x
+y <- y + rnorm(length(x), sd = 0.1)
+
+ggplot2::ggplot(data.frame(x = x, y = y), ggplot2::aes(x = x, y = y)) +
+  ggplot2::geom_point() + 
+  ggplot2::theme_bw() + 
+  ggplot2::geom_smooth(formula = y ~ x, method = "lm", se = FALSE) +
+  ggplot2::annotate("point", x = 0.95, y = 0.3, color = "orange", size = 2)
+
+@
+
+\begin{enumerate}[a)]
+  \item Consider the above linear regression problem. How will the model 
```

Comment:
"Describe how the model parameters.." => Wann immer möglich, sind Aufforderungen besser als Fragen.